web/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -233,6 +234,7 @@ class Panel extends Component<PanelProps, PanelState> {
               onExpressionChange={this.handleExpressionChange}
               executeQuery={this.executeQuery}
               loading={this.state.loading}
+              enable={this.props.enableMetricAutocomplete}
```

Comment:
If I just read `enable` in the context of `ExpressionInput`, I wouldn't know what it means, as it lacks specificity / context. Could you give it a longer name here too?

Reply:
Went ahead and used the full name used elsewhere.